keyboards/keebio/iris/keymaps/olligranlund_nordic_v2/keymap.c

```diff
@@ -0,0 +1,126 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
+#define _ADJUST 16
```

Comment:
Should use an enum for this:
```suggestion
enum layers (
    _QWERTY,
    _LOWER,
    _RAISE
    _ADJUST,
};
```